routesrv/eskipbytes.go

```diff
@@ -72,6 +72,7 @@ func (e *eskipBytes) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if initialized {
 		w.Header().Add("Etag", etag)
 		w.Header().Add("Content-Type", "text/plain; charset=utf-8")
+		w.Header().Add("Content-Encoding", "gzip")
```

Comment:
It is not enough to add a header to serve gzip - the response body should be compressed as well, see https://stackoverflow.com/questions/14073393/serving-gzipped-content-for-go

In general the compression trades network consumption for cpu usage on both server and client sides. We should justify this change somehow, e.g. by showing how much it will compress.

Reply:
Maybe add the compress filter in the route chain to serve the content to re-use skipper's feature set. It's of course a little more work.